sql/20190424173819_create_jobs_table.sql

```diff
@@ -0,0 +1,13 @@
+-- migrate:up
+CREATE TABLE "jobs" (
+    "id"        SERIAL  NOT NULL,
+    "name"      TEXT    NOT NULL,
+    "wage"      INT     NOT NULL,
+    "client_id" INT     NOT NULL,
+    PRIMARY KEY (id),
+    FOREIGN KEY (client_id) REFERENCES clients (id) ON DELETE CASCADE ON UPDATE CASCADE,
+    UNIQUE ("name", "wage", "client_id")
```

Comment:
id must be first in UK, also I suggest dont use such UK. Or, may be it is a good idea to use only (client_d, name) fields for UK

Reply:
actually person can work at one company on two positions with the different salary